examples/mugen/test/retrieval/test_video_clip.py

```diff
@@ -72,8 +77,14 @@ def test_forward_pretrained(self, utils):
         assert_expected(
             actual=out.sum(), expected=torch.as_tensor(expected_sum), rtol=0, atol=1e-4
         )
+        assert next(encoder.parameters()).requires_grad
```

Comment:
Think about whether we indeed need two knobs (pretrained, trainable). Do they really serve the user? If not, we could probably simplify our API as well as reduce complexity in unit tests.

Reply:
resolving since we can continue discussion in the comment above